changelog.d/14053.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug introduced in Synapse v1.53.0 when querying `/publicRooms` with both a `room_type` filter and a `third_party_instance_id`.
```

Comment:
```suggestion
Fix a bug introduced in Synapse 1.53.0 when querying `/publicRooms` with both a `room_type` filter and a `third_party_instance_id`.
```

The convention is apparently to omit the `v` when prefixing a version number with "Synapse", e.g. `Synapse 1.2.3`; reserve v for standalone versions like v`1.2.3`.

Reply:
We seem wildly inconsistent here. 😢